JSDemos/Demos/DataGrid/FilteringAPI/description.md

```diff
@@ -1 +1,15 @@
-The **DataGrid** widget ships with an&nbsp;integrated filtering API, allowing you apply filters in&nbsp;code. In&nbsp;this example, the grid&rsquo;s filter function is&nbsp;called to&nbsp;filter data based on&nbsp;selections made within the drop-down menu above the **DataGrid**.
\ No newline at end of file
+**DataGrid** includes the following API that you can use to filter data:     
+
+- [filter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#filter) / [filter(filterExpr)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#filterfilterExpr)       
+Gets or sets a filter expression for the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource).
+
+- [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue)       
+A filter expression applied using the following UI elements: [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filter builder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder).
+
+- [getCombinedFilter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getCombinedFilter) / [getCombinedFilter(returnDataField)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getCombinedFilterreturnDataField)       
+Gets the total filter that includes filters from the UI elements and the filter applied to the **dataSource**. If you specify the `returnDataField` parameter, the return value will contain data field names instead of [getters](/Documentation/Guide/Data_Binding/Data_Layer/#Getters_And_Setters).
+
+- [clearFilter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#clearFilter) / [clearFilter(filterName)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#clearFilterfilterName)       
+Discards all filters applied to the widget. If you specify the `filterName` parameter, this method discards all filters of a specific type.
+
+In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. When you select an item from the **SelectBox**, the **filter(filterExpr)** or **clearFilter()** method is called, depending on your selection.
```

Comment:
```suggestion
In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. When you select an item from the **SelectBox**, the **filter(filterExpr)** or **clearFilter()** method is called.
```

Reply:
Why add 'depending on your selection' if the selection options are known?

Reply:
The reason is we have six selection options, and only one of them calls the **clearFilter()** method. Other selection options call the **filter(filterExpr)** method is called. Do you think that it will be clear for customers without the phrase that you suggest to omit?

Reply:
Is it clear from the names which method does what? For instance, if it is called 'clear' something, you know it corresponds to the clearFilter() method.

Reply:
No, it isn't clear and this is the problem. The selection options are the following:
![image](https://user-images.githubusercontent.com/33040934/92885310-507ec100-f41b-11ea-861f-f8e5f01c36b3.png)
Only the first option ('All') calls the **clearFilter()** method. Other options call the **filter(filterExpr)** method.

Reply:
Ok, but then you should say that: In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. 'All' calls the clearFilter() method and the other items call the **filter(filterExpr)**method.